en/02_Developer_Guides/06_Testing/00_Unit_Testing.md

```diff
@@ -39,44 +39,40 @@ class PageTest extends SapphireTest
 }
 ```
 
-[info]
-Tests for your application should be stored in the `app/tests` directory. Test cases for add-ons should be stored in
-the `(modulename)/tests` directory.
-
-Test case classes should end with `Test` (e.g. `PageTest`) and test methods must start with `test` (e.g. `testMyMethod`).
-
-Ensure you [import](http://php.net/manual/en/language.namespaces.importing.php#example-252) any classes you need for the test, including `SilverStripe\Dev\SapphireTest` or `SilverStripe\Dev\FunctionalTest`.
-[/info]
+> [!NOTE]
+> Tests for your application should be stored in the `app/tests` directory. Test cases for add-ons should be stored in
+> the `(modulename)/tests` directory.
+>
+> Test case classes should end with `Test` (e.g. `PageTest`) and test methods must start with `test` (e.g. `testMyMethod`).
+>
+> Ensure you [import](http://php.net/manual/en/language.namespaces.importing.php#example-252) any classes you need for the test, including `SilverStripe\Dev\SapphireTest` or `SilverStripe\Dev\FunctionalTest`.
 
 A Silverstripe CMS unit test is created by extending one of two classes, [SapphireTest](api:SilverStripe\Dev\SapphireTest) or [FunctionalTest](api:SilverStripe\Dev\FunctionalTest).
 
 [SapphireTest](api:SilverStripe\Dev\SapphireTest) is used to test your model logic (such as a `DataObject`), and [FunctionalTest](api:SilverStripe\Dev\FunctionalTest) is used when
 you want to test a `Controller`, `Form` or anything that requires a web page.
 
-[info]
-`FunctionalTest` is a subclass of `SapphireTest` so will inherit all of the behaviors. By subclassing `FunctionalTest`
-you gain the ability to load and test web pages on the site.
-
-`SapphireTest` in turn, extends `PHPUnit\Framework\TestCase`. For more information on `PHPUnit\Framework\TestCase` see
-the [PHPUnit](http://www.phpunit.de) documentation. It provides a lot of fundamental concepts that we build on in this
-documentation.
-[/info]
+> [!NOTE]
+> `FunctionalTest` is a subclass of `SapphireTest` so will inherit all of the behaviors. By subclassing `FunctionalTest`
+> you gain the ability to load and test web pages on the site.
+>
+> `SapphireTest` in turn, extends `PHPUnit\Framework\TestCase`. For more information on `PHPUnit\Framework\TestCase` see
+> the [PHPUnit](http://www.phpunit.de) documentation. It provides a lot of fundamental concepts that we build on in this
+> documentation.
 
 ## Test databases and fixtures
 
+> [!CAUTION]
+> As the test runner will create new databases for the tests to run, the database user should have the appropriate
+> permissions to create new databases on your server.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.